### ISSUE TITLE
Creative Fly

### ISSUE THREAD

**Issue by Shohid-Nunu** [MeteorDevelopment/meteor-client#4233]
### Describe the feature

Hi, just wanted to suggest a module/mode which is present in other clients known as creative fly. The devs may add a mode in flight as creative fly which enables the player to fly in vanilla mode where the player can choose to fly by double clicking the spacebar and also fly down and walk normally after flying down to the ground (basically flying in creative mode). Bless you devs 

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by Wide-Cat** [comment]
I guess we could add a setting to flight to mimic creative behaviour by toggling with a double spacebar

**Comment by Shohid-Nunu** [comment]
> I guess we could add a setting to flight to mimic creative behaviour by toggling with a double spacebar

yeah but I really dislike the flight modes as flying up lifts the player really high but wurst client has another module as creativefly idk how it works but even bedrock clients have that feature such as horion so I think adding this module would be a cakewalk for the devs
